Added  particular file name (#5729)
MicrosoftDocs/windows-itpro-docs: devices/surface/microsoft-surface-brightness-control.md

Brightness Control app.

> Available for download with [Surface Tools for

Reviewer: ```suggestion
Available for download with [Surface Tools for IT](https://www.microsoft.com/download/details.aspx?id=46703),
```

> Available for download with [Surface Tools for

Reviewer 2: ```suggestion
Available for download with [Surface Tools for IT](https://www.microsoft.com/download/details.aspx?id=46703).
```

> IT](https://www.microsoft.com/download/details.aspx?id=46703), Download only this file **Surface_Brightness_Control_v1.16.137.0.msi**. Surface Brightness Control is designed to help reduce thermal load and lower the overall carbon

Reviewer: ```suggestion
Surface Brightness Control is designed to help reduce thermal load and lower the overall carbon footprint for deployed Surface devices.
If your plan is only to get this tool from the download page, select the file **Surface_Brightness_Control_v1.16.137.0.msi** in the available list.
```

> footprint for deployed Surface devices. The tool automatically dims the screen when not in use and

Reviewer: ```suggestion
The tool automatically dims the screen when not in use and includes the following configuration options:
```

> includes the following configuration options:

Reviewer: ```suggestion
```
